web/app.php

```diff
@@ -1,10 +1,18 @@
 <?php
-use LibraryLoaderAutoloader,
-    LibraryControllerFrontController;
-    
-require_once __DIR__ . "/Library/Loader/Autoloader.php";
-$autoloader = new Autoloader;
-$autoloader->register();
-
-$frontController = new FrontController();
-$frontController->run();
\ No newline at end of file
+
+require_once __DIR__ . './../vendor' . '/autoload.php';
+require_once __DIR__ . './../app/config' . '/services.php';
```

Comment:
Is het altijd de bedoeling dat je services.php requires in the app.php. Kan dit niet door autoload worden opgepakt?